spec/actionizer_spec.rb

```diff
@@ -175,6 +175,29 @@ def self.call(_params, _arg2)
         expect(result).to be_failure
         expect(result.error).to eq('inner error')
       end
+
+      context 'and error response field begins with "error" but has additional characters' do
+        let(:failure_action_class) do
+          Class.new do
+            include Actionizer
+            def call
+              fail!(errors_xxxxx: 'inner error')
+            end
+          end
+        end
+
+        it 'calls fail! and passes on result.errors_xxxxx and skips calling the second class' do
+          expect(failure_action_class).to receive(:call)
+                                            .with(foo: 'bar').once.and_call_original
```

Comment:
I would suggest you just put this on one line. Rubocop should be good with that.